### PR TITLE
add devopsian blog

### PR DESCRIPTION
**Pull request by canidam**

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -979,5 +979,22 @@ module.exports = [
       'Node.js',
       'Web Development',
     ]
+  },
+  {
+    name: 'Chen A',
+    description: 'Just another blog about Devops/SRE - blog about infrastructure, software-engineering, et al.',
+    url: 'https://devopsian.net',
+    tags: [
```
Comment (Dawntraoz) [inline data.js:987]: Do you have a twitter account? I think the handle is currently needed { twitter: '@', }.

Or can we hide it when they don't have one @MarkoDenic ?

Comment (canidam) [reply data.js:987]: No twitter account :)

Comment (canidam) [reply data.js:987]: can't we merge it without a twitter account?

**Review by MarkoDenic — APPROVED** [review]

**Comment by MarkoDenic** [comment]
Thanks for your patience, @canidam. 
@Dawntraoz, I added a small patch to omit the Twitter link, if it's not provided. Thanks for your assistance.

**Pull request merged**